src/resmom_win/win/mom_mach.c

```diff
@@ -576,7 +576,7 @@ mem_sum(job *pjob)
 			JobObjectBasicProcessIdList,
 			pProcessList, pidlistsize, NULL))
 			log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__,
-				"QueryInformationJobObject for %d failed to get processe ID list with error %lu", pjob->ji_hJob, GetLastError());
+				"QueryInformationJobObject for job %d, handle %d failed to get processe ID list with error %lu", pjob->ji_qs.ji_jobid, pjob->ji_hJob, GetLastError());
```

Comment:
change your format specifier here and some other places as well wherever you are trying to log jobid